pkg/config/command.go

```diff
@@ -185,3 +191,42 @@ func GenFormatCmd(secrets map[string]string, noUpdate bool, setting *JfsSetting)
 
 	return
 }
+
+// GenJfsVolUUID get UUID from result of `juicefs status <volumeName>`
+func GenJfsVolUUID(ctx context.Context, s *JfsSetting) (string, error) {
```

Comment:
```suggestion
func GetJfsVolUUID(ctx context.Context, s *JfsSetting) (string, error) {
```

It looks like `get` rather than `gen`?